src/euphorie/client/browser/webhelpers.py

```diff
@@ -653,6 +653,11 @@ def getSecret(self):
     def get_current_account(self):
         return get_current_account()
 
+    @memoize_contextless
+    def get_my_account_id(self):
+        account = self.get_current_account()
+        return account and account.id or ""
```

Comment:
Just a suggestion: there is no need for introducing a new function given that in the pt you can do 
```
tal:define="account webhelpers/get_current_account" 
value="${account/id|nothing}"
```
It is a pity that we do not have an account property otherwise:
```
value="${webhelpers/account/id|nothing}"
```
was better.

Reply:
Thanks! Will use your suggestion